package.json

```diff
@@ -43,6 +43,7 @@
     "@fortawesome/fontawesome-svg-core": "^6.5.1",
     "@libsql/client": "^0.5.5",
     "@shopify/draggable": "^1.1.1",
+    "@square/svelte-store": "square/svelte-store",
```

Comment:
This library has a store named `persisted`, which wraps local storage and other types of "simple" browser storage as well. Related issue: #50.